packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -49,6 +49,22 @@ const double _kMinButtonSize = kMinInteractiveDimension;
 /// ** See code in examples/api/lib/material/icon_button/icon_button.0.dart **
 /// {@end-tool}
 ///
+/// ### Icon sizes
+///
+/// When creating an icon button with an [Icon], do not override the icon's size
```

Comment:
there's research that shows that when you tell people not to do something, they remember the opposite.

for this reason, flutter's documentation voice usually uses "avoid" rather than "do not"

as in, "When ..., avoid overriding..."